src/pages/404.tsx

```diff
@@ -1,28 +1,19 @@
 import React from "react";
-import { PageProps } from "gatsby";
-
-import fig404 from "../../static/404.png";
+import { IcLink, IcEmptyState, IcSectionContainer } from "@ukic/react";
 
-import "./404.css";
+import { PageProps } from "gatsby";
 
 const NotFoundPage: React.FC<PageProps> = () => (
-  <ic-section-container aligned="full-width" id="container-404">
-    <img src={fig404} alt="" width="200" loading="lazy" />
-    <ic-typography variant="h1" apply-vertical-margins>
-      <h2>That page wasn&apos;t found</h2>
-    </ic-typography>
-    <ic-typography variant="body" apply-vertical-margins>
-      <p>
-        Sorry, we can&apos;t find the page you&apos;re looking for. This may be
-        because the page has moved or there was a typo in the URL.
-      </p>
-    </ic-typography>
-    <ic-typography variant="label" apply-vertical-margins>
-      <p>
-        Try <ic-link href="/">searching from the homepage</ic-link>.
-      </p>
-    </ic-typography>
-  </ic-section-container>
+  <IcSectionContainer aligned="full-width">
+    <IcEmptyState
```

Comment:
The empty state component can display an image - could we keep the original one? I think it looked quite good with that image but just my opinion :)

Reply:
Image re added :)